locales/en/messages.json

```diff
@@ -5020,6 +5020,13 @@
     "osdWarningRssiDbm": {
         "message": "Warns when RSSI dBm value is below alarm setting"
     },
+    "osdWarningTextOverCap": {
+        "message": "mAh capacity exceeded",
+        "description": "One of the warnings that can be selected to be shown in the OSD"
+    },
```

Comment:
One thing more, to discuss... now the OSD elements (including the warnings) are ordered by alphabetical order. I think is better to include this warning near the other battery warnings. Maybe with something like:
- "Battery mAh capacity exceeded" or maybe "Battery: mAh capacity exceeded"

What do you think?

Reply:
Might be clearer but it's a bit too long compared to the other warning labels. How about "Battery over capacity"? Not too long and still descriptive. If you prefer "Battery mAh capacity exceeded" despite the length I'll add that

Reply:
Let's wait for other thoughts... @mikeller what do you think?

Reply:
'Battery capacity exceeded' or 'Battery over capacity' sounds about right - the units used to measure this are not really relevant anyway.

Reply:
"Battery over capacity" it is 👍